hbase-compression/hbase-compression-zstd/src/main/java/org/apache/hadoop/hbase/io/compress/zstd/ZstdCodec.java

```diff
@@ -51,6 +60,9 @@ public class ZstdCodec implements Configurable, CompressionCodec, ByteBuffDecomp
   public static final int ZSTD_BUFFER_SIZE_DEFAULT = 256 * 1024;
   public static final String ZSTD_DICTIONARY_KEY = "hbase.io.compress.zstd.dictionary";
 
+  private static final Cache<String, Pair<ZstdDictDecompress, Integer>> DECOMPRESS_DICT_CACHE =
+    CacheBuilder.newBuilder().maximumSize(100).expireAfterAccess(10, TimeUnit.MINUTES).build();
```

Comment:
Should the details of this cache be user-configurable?

Reply:
My preference is for less low-value configuration options. But, I'm happy to do it either way, what do you prefer?

Reply:
Nope, I'm fine with leaving them as hard-coded for now. Just asking the question to prompt the discussion.